projects/plugins/jetpack/modules/masterbar/nudges/additional-css/class-wporg-additional-css-manager.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * WPORG_Additional_CSS_Manager file
+ *
+ * Responsible with replacing the Core Additional CSS section with an upgrade nudge on Atomic.
+ *
+ * @package Jetpack
+ */
+
+namespace Automattic\Jetpack\Dashboard_Customizations;
+
+/**
+ * Class WPORG_Disable_Additional_CSS
+ *
+ * @package Automattic\Jetpack\Dashboard_Customizations
+ */
+class WPORG_Additional_CSS_Manager {
```

Comment:
I think renaming this to `Atomic_Additional_CSS_Manager` makes more sense, since this cannot be used on WP.org sites (e.g. the "Business plan" terminology or the checkout URL are only valid for WP.com sites).